indico/modules/events/sessions/templates/management/_session_list.html

```diff
@@ -0,0 +1,108 @@
+                    <th class="i-table">{% trans %}Colour{% endtrans %}</th>
+                    <th class="i-table" data-sorter="false">{% trans %}Blocks{% endtrans %}</th>
+                    <th data-sorter="false"></th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for session in sessions -%}
+                    <tr class="i-table">
+                        <td class="i-table">
+                            <input class="select-row" type="checkbox" value="{{ session.id }}">
+                        </td>
+                        <td class="i-table">#{{ session.friendly_id }}</td>
+                        <td class="i-table"><strong>{{ session.title }}</strong></td>
+                        <td class="i-table">{{ session.code }}</td>
+                        <td class="i-table">
+                            {%- if session.is_poster %}
+                                {%- trans %}Poster{% endtrans -%}
+                            {% else %}
+                                {%- trans %}Standard{% endtrans -%}
+                            {% endif -%}
+                        </td>
+                        <td class="i-table">
+                            <div class="group">
+                                <a class="i-button icon-palette palette-picker-trigger"
+                                   style="background: #{{ session.background_color }};
+                                          color: #{{ session.text_color }} !important;"
+                                   data-method="PATCH" data-href="{{ url_for('.session_rest', session) }}"
+                                   data-colors="{{ default_colors | tojson | forceescape }}">
+                                </a>
+                            </div>
+                        </td>
+                        <td class="i-table">
+                            <span class="i-label show-session-blocks">
+                                {%- trans %}Session blocks {% endtrans -%}
+                                <span class="badge">{{ session.blocks | length }}</span>
+                            </span>
+                        </td>
+                        <td class="i-table">
+                            <div class="group right">
+                                <a class="icon-edit highlight edit-session action-icon"
+                                   title="{% trans %}Edit this session{% endtrans %}"
+                                   data-title="{% trans title=session.title %}Edit session {{ title }}{% endtrans %}"
+                                   data-href="{{ url_for('.modify_session', session) }}"
+                                   data-update=".sessions-wrapper"
+                                   data-ajax-dialog></a>
+                                <a class="icon-time highlight action-icon"
+                                   title="{% trans %}Go to this session timetable{% endtrans %}"
+                                   data-href="#"
```

Comment:
I don't think this will use `data-href` etc; it'll most likely be a normal link (so I'd use `href="#"` for now)